### ISSUE TITLE
Quest items

### ISSUE THREAD

**Issue by Jerntaw** [Questie/Questie#5243]
Hello, tell me, is it possible to make it so that the item shows that it is for a quest, although this quest has not been taken at the moment, but can be taken in the future?

**Comment by MyTechnoHunter** [comment]
its.. complicated.. all the things addon can show quest items.. _we_ can show quest items and by that i mean items tagged as Quest (normally quest starters) but i presume you are asking about item x being needs for cooking quest y that you dont have yet?

if not clarification needed

**Comment by Jerntaw** [comment]
Yes, there are no quests yet, but the item is knocked out, and it may be useful in the future for a quest, and you sell it to the vendor, unaware of this

**Comment by MyTechnoHunter** [comment]
there were a few addons that did this (show in tooltip item used for blaat) but the problem is it takes a DB of quests and items, which isnt in the classic API yet.. retail API has a complete set of server pulled info from in game for recipes and reagents for same.. 

so, much as i personally would love it, i dont see it happening in Questie at this time.. feel free to tag this as a feature request and we can keep it in mind for the down time (yeah, i sniggered while writing this)
